clouddriver-kubernetes-v2/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesV2SecurityGroupProvider.java

```diff
@@ -144,4 +144,9 @@ public KubernetesV2SecurityGroup get(
         .findFirst()
         .orElse(null);
   }
+
+  @Override
+  public KubernetesV2SecurityGroup getById(String account, String region, String id, String vpcId) {
+    throw new UnsupportedOperationException("Not currently implemented.");
```

Comment:
I took the liberty of implementing the overrides for all sub-classes, since they seemed straightforward, but not this one, for the opposite reason.

Reply:
cc @maggieneterval @ezimanyi

Reply:
BTW, I didn't think this was of any concern at this time since the controller change is effectively exposing a new API for all practical purposes (meaning no one would have been using it), but would definitely love a contribution to fill in this gap!

Reply:
Thanks for the heads up! Is keel the only anticipated client of this new API (for now)?

Reply:
Yes, it is.